client/src/components/Sidebar/index.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable react/jsx-wrap-multilines */
+import React, { useState, useEffect } from 'react';
+import { Layout, Menu, Icon } from 'antd';
+
+const { SubMenu } = Menu;
+const { Sider } = Layout;
+
+const Sidebar = () => {
+  const [userChannels, setUserChannels] = useState([]);
+  const [groupChannels, setGroupChannels] = useState([]);
+  useEffect(() => {
+    setGroupChannels([
+      { name: 'channel1', id: '123123' },
+      { name: 'channel2', id: '123122' },
+    ]);
+    setUserChannels([
+      { name: 'Person1', id: '1223' },
+      { name: 'Person2', id: '1122' },
+    ]);
+  }, []);
+  function switchToChannel(channelId) {
```

Comment:
move this function out of sideBar component, as this will be created for every render.

Reply:
Removed for now.